src/action-creators/cursorDown.js

```diff
@@ -8,18 +8,17 @@ import {
   isDivider,
   restoreSelection,
   selectNextEditable,
+  getThoughts
 } from '../util.js'
 
 export const cursorDown = ({ target }) => dispatch => {
-
   const { cursor } = store.getState()
-
   if (cursor) {
     // select next editable
     const nextThought = getThoughtAfter(cursor)
-
+    const children = getThoughts(cursor)
```

Comment:
It would be better to place this line of code in the `if` scope. Then it doesn't get called unnecessarily when `nextThought` is `null`.